src/chapter2/01-html-and-the-dom.md

```diff
@@ -0,0 +1,139 @@
+## HTML and the DOM
+
+<div style="text-align: right"> Hey guys, how do I parse HTML with a regular expression? <br> <i> - Famous last words </i> </div>
+
+### A minimal HTML file
+
+Inside your project directory (*easy-opus* in our case) create a directory which will contain the (website) client. Give the directory a descriptive name (for example *client*). If you followed our naming scheme, your directory structure should now look as follows:
+
+```
+easy-opus/
+└─── client/
+```
+
+Let's write our first webpage. We do so using **HTML** which is a *markup language* for defining for documents that should be displayed in a browser.
+
+> A markup language specifies rules which define what kind of information can be included in a document. Note that markup languages and programming languages are different things. A markup language essentially allows you to create documents to be displayed, while programming languages are much more powerful than that.
+
+Navigate to the client directory and create a file named `index.html` in there. Now write some minimal useful HTML to display a bunch of tasks:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Easy Opus</title>
+  </head>
+  <body>
+    <div id="root">
+      <div id="app">
+        <h1 id="headline">My tasks</h1>
+        <div id="taskList">
+          <p id="task1">
+            Read the MERN book: Read and understand the MERN book.
+          </p>
+          <p id="task2">
+            Write a website: Create a new and cool website.
+          </p>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
+```
```

Comment:
afaik default prettier does not indent children of `<html>`

Reply:
We'll should automate formatting codeblocks

Reply:
I'll take care of it